stat/samplemv/samplemv.go

```diff
@@ -126,16 +126,16 @@ func latinHypercube(batch *mat.Dense, q distmv.Quantiler, src *rand.Rand) {
 // a good proposal distribution will bound this sampling weight. This implies the
 // support of q(x) should be at least as broad as p(x), and q(x) should be "fatter tailed"
 // than p(x).
-//
-// If weights is nil, the weights are not stored. The length of weights must equal
-// the length of batch, otherwise Importance will panic.
 type Importance struct {
 	Target   distmv.LogProber
 	Proposal distmv.RandLogProber
 }
 
 // SampleWeighted generates rows(batch) samples using the Importance sampling
 // generation procedure.
+//
+// If weights is nil, the weights are not stored. The length of weights must equal
```

Comment:
"Weights are not stored" seems a weird way to say this. Also, please add to the second sentence that this is only when weights is not nil.